astropy/modeling/fitting.py

```diff
@@ -341,6 +341,118 @@ def __call__(self, model, x, y, z=None, weights=None, rcond=None):
         return model_copy
 
 
+class FittingWithOutlierRemoval(object):
+    """
+    This class combines an outlier removal technique with a fitting procedure.
+    Basically, given a number of iterations `niter`, outliers are removed in
+    and fitting is performed for each iteration.
+    """
+
+    def __init__(self, fitter, outlier_func, niter=3, **outlier_kwargs):
+        """
+        Parameters
+        ----------
+        fitter : An Astropy fitter
+            An instance of any Astropy fitter, i.e., LinearLSQFitter,
+            LevMarLSQFitter, SLSQPLSQFitter, SimplexLSQFitter, JointFitter.
+        outlier_func : function
+            A function for outlier removal.
+        niter : int (optional)
+            Number of iterations.
+        outlier_kwargs : dict (optional)
+            Keyword arguments for outlier_func.
```

Comment:
Move the `Parameters` section to the class docstrings, just like the rest of the classes.